circuitbreaker/umbrel-app.yml

```diff
@@ -0,0 +1,56 @@
+manifestVersion: 1
+id: circuitbreaker
+category: Finance
+name: Circuit Breaker
+version: "v0.4.0"
+tagline: Circuit Breaker is to Lightning what firewalls are to the internet.
+description: >-
+  It allows nodes to protect themselves from being flooded with htlcs. 
+  With circuitbreaker a maximum to the number of in-flight htlcs can be set on a per-peer basis.
+  Known and trusted peers for example can be assigned a higher maximum, while a new channel from a previously unseen node may be limited to only a few pending htlcs.
+  Furthermore it is possible to apply rate limits to the number of forwarded htlcs. 
+  This offers protection against DoS/spam attacks that rely on large numbers of fast-resolving htlcs. 
+  Rate limiting is implemented with a Token bucket. 
+  Via the UI the minimum interval between htlcs can be specified.
+  Large numbers of htlcs are also required for probing channel balances. 
+  Reducing the information leakage through probing could be another reason to put in place a rate limit for untrusted peers.
+  
+
+  Operating modes
+
+
+  - There are multiple modes in which circuitbreaker can operate. A default mode and per-peer overrides can be configured via the web ui.
+
+  - fail: Fail back htlcs when limits are exceeded. This minimizes the lock-up of liquidity on the incoming side, but does affect your reputation as a routing node.
+
+  - queue: Queue htlcs when limits are exceeded. Items are popped from the queue when the number of pending htlcs is below the maximum and the rate limit allows another forward. This mode penalizes upstream nodes for the bad traffic that the deliver by locking up liquidity along the route. This may push upstream nodes to install a firewall too and constrain the mishaving node.
+
+  Even in case of circuit breaker malfunction, queue mode should never cause channel force closes because of lnd's built-in protection that auto-fails htlcs that aren't resolved.
+
+  - WARNING: Auto-fail is not yet released and scheduled for lnd 0.16. With earlier lnd versions, you risk force-closes!
+
+  - queue_peer_initiated: This mode is also queuing htlcs, but only those that come in through channels for which we aren't the channel open initiator. Not being the initiator means that the remote node is carrying the cost of a potential force-closure with stuck htlcs. For channels that we initiated, the safer fail mode is used.
+
+  - WARNING: See queue mode warning.
+  
+releaseNotes: >-
+  - Initial release on the umbrel app store
+  
+developer: Joost Jager
+website: https://github.com/joostjager
+dependencies: 
+  - lightning
+repo: https://github.com/lightningequipment/circuitbreaker
+support: <tbd>
```

Comment:
I think we can just link https://github.com/lightningequipment/circuitbreaker/issues for now?